docs/api/java/getting-started.md

```diff
@@ -81,23 +73,23 @@ import org.web3j.crypto.Credentials;
 
 public class Main {
     public static void main(String ...args) {
-        long chainId = 123L;// Chainid of the zksync network
+        long chainId = 123L;// Chainid of the ZkSync Era network
```

Comment:
please mention the chain ids for testnet and mainnet in a tip right below this script